### PR TITLE
Hamburglar

### PR DESCRIPTION
**Pull request by lisagumerman**

### CODE REVIEW

File: aire/src/button/button.pug [hunk 1]
```diff
@@ -1,2 +1,4 @@
 template
-    button.uk-button(ref="button" checked.bind="selected") ${label}
\ No newline at end of file
+    button.uk-button(ref="button")
+        span(if.bind="icon" class.bind="icon")
+        | ${label}
```
Comment (josiahhaswell) [inline aire/src/button/button.pug]: Why not keep it on the same line?

File: aire/src/button/button.ts [hunk 2]
```diff
@@ -8,6 +8,9 @@ export class AireButton {
   @bindable
   label     : string;
 
+  @bindable
+  icon      : string;
```
Comment (josiahhaswell) [inline aire/src/button/button.ts]: This is ok for icon-classes but what if we need an image URL?

File: aire/src/navbar/navbar.pug [hunk 3]
```diff
@@ -1,3 +1,3 @@
 template
-    nav.uk-navbar-container.uk-navbar-sticky.uk-dark(ref="element")
+    nav.uk-navbar-container.uk-navbar-sticky.uk-dark(ref="element" uk-navbar)
```
Comment (josiahhaswell) [inline aire/src/navbar/navbar.pug]: these pseudo-elements are only used when initializing the entire page, which we don't want to do.

File: aire/src/offcanvas/offcanvas.pug [hunk 4]
```diff
@@ -1,5 +1,5 @@
 template
-    .aire.uk-offcanvas(ref="element")
+    .aire.uk-offcanvas(ref="element" uk-offcanvas)
```
Comment (josiahhaswell) [inline aire/src/offcanvas/offcanvas.pug]: ditto

File: aire/src/offcanvas/offcanvas.ts [hunk 5]
```diff
@@ -1,14 +1,38 @@
...
+      this.offcanvas.hide();
+
+    }
+    else {
+      this.offcanvas.show();
+    }
+    this.open = !this.open;
+  }
+
+  show() : void {
```
Comment (josiahhaswell) [inline aire/src/offcanvas/offcanvas.ts]: can we get an equivalent hide() method?

File: demo/src/index.pug [hunk 6]
```diff
@@ -2,13 +2,15 @@ template
     aire-application
         aire-header(primary)
             navbar-section(left)
+                navbar-item
+                    aire-button(icon="fa fa-bars" square click.delegate="toggleOffCanvas()")
+            navbar-section(center)
                 navbar-item(icon="dist/icons/logo.svg" large logo)
-                navbar-item(logo).pull-left Aurelia-UIKit
```
Comment (josiahhaswell) [inline demo/src/index.pug]: thanks for fixing that hack

File: aire/src/button/button.pug (hunk 1 above)
Comment (lisagumerman) [reply aire/src/button/button.pug]: cause it doesn't go in the span!

File: aire/src/button/button.ts (hunk 2 above)
Comment (lisagumerman) [reply aire/src/button/button.ts]: yeah, that's a bit of a todo to go back to -- I noticed we're missing it in some other places where we _only_ allow for svgs, and I think icon font is more likely.

File: aire/src/offcanvas/offcanvas.ts (hunk 5 above)
Comment (lisagumerman) [reply aire/src/offcanvas/offcanvas.ts]: never!

**Pull request merged**